src/Main.elm

```diff
@@ -125,21 +125,26 @@ stepSpawnState config { kurvesLeft, alreadySpawnedKurves, ticksLeft } =
 
 
 update : Msg -> Model -> ( Model, Cmd Msg )
-update msg ({ config, pressedButtons } as model) =
+update msg model =
+    updatePure msg model |> Tuple.mapSecond makeCmd
+
+
+updatePure : Msg -> Model -> ( Model, Maybe CanvasAction )
+updatePure msg ({ config, pressedButtons } as model) =
```

Comment:
You can also call `updatePure` just `update`, and name the other one `updateWrapper` or something. I’ve done that before, since I’m used to jumping to a function called `update` when I want to do update logic.

Reply:
Obfuscation of the day! 😂

Reply:
All joking aside, this was a great suggestion. `update` is now back, and the `Cmd` is constructed in `updateWithCmd` instead.